Applications/VGG/jni/main.cpp

```diff
@@ -20,6 +20,9 @@
 
 #include <cifar_dataloader.h>
 #include <model.h>
+#ifdef PROFILE
+#include <profiler.h> // disable including this in android as profiler is not exposed yet to devel header
+#endif
```

Comment:
You may have
```
#ifdef PROFILE
#define PROFILE_BEGIN auto listener = std::make....
#define PROFILE_END std::cout << *listener;
#else
#define PROFILE_BEGIN do { } while (0)
#define PROFILE_END do { } while (0)
#endif
```
in ```profiler.h``` and remove #ifdef/#endif from .cpp files.

Reply:
we might need `listener in BEGIN and END as an parameter. That means that we need the #ifdef as well.
``` c++
#ifdef PROFILE
auto listener = std::make_shared<nntrainer::profile::GenericProfileListener>();
PROFILE_BEGIN(listener);
#endif
....
PROFILE_END(listener);
```
Otherwise, there is compile error saying "listener is not defined" in PROFILE_END.